freeipa/src/main/java/com/sequenceiq/freeipa/service/resource/ResourceAttributeUtil.java

```diff
@@ -1,25 +1,42 @@
 package com.sequenceiq.freeipa.service.resource;
 
 import java.io.IOException;
+import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
-import com.sequenceiq.cloudbreak.common.json.Json;
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.model.VolumeSetAttributes;
 import com.sequenceiq.cloudbreak.common.exception.CloudbreakServiceException;
+import com.sequenceiq.cloudbreak.common.json.Json;
 import com.sequenceiq.freeipa.entity.Resource;
 
 @Component
 public class ResourceAttributeUtil {
 
-    public <T> Optional<T> getTypedAttributes(Resource resource, Class<T> attributeType) {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ResourceAttributeUtil.class);
+
+    public <T> Optional<T> getTypedAttributes(Resource resource) {
```

Comment:
Not a big deal but I do not think the type parameter `<T>` brings any benefit here. It is not bound to either any method args or the parent class.